test.js

```diff
@@ -1,6 +1,12 @@
 var expect = require('chai').expect;
 var Plugin = require('./');
 var Handlebars = require('handlebars');
+var sysPath = require('path');
+var sysFile = require('fs');
+ 
+var nsPath = sysPath.join(__dirname, 'ns.js'); 
+var nsFile = sysFile.readFileSync(nsPath, {encoding : 'utf8' });
+eval(nsFile);
```

Comment:
Hmm, I guess there's no better way to do this, is there?

Please add some comments explaining what's happening here and why it's being done like this.